website/docs/components/application-state/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,25 @@
+## Usage
+
+### When to use
+- When a user's action does not yield any results due to incorrect or incomplete input.
+- When an application encounters an issue or error during its operation.
+
+### When not to use
+- When the absence of content is expected and does not require an explanation to the user. 
+- When there is a clear and intuitive way to add or populate content. 
+
+## Icon
+
+- The icon should be relevant and communicate the same purpose as the errorCode.
+
+## Actions
+
+- In the footer, you can include two stand-alone links. 
```

Comment:
```suggestion
- In the footer, you can include up to two stand-alone links. 
```

@andgen404 I wonder if we want to reword this just a little bit. WDYT?

Reply:
@MelSumner I agree with the change